crates/cargo-contract/src/cmd/storage.rs

```diff
@@ -65,6 +70,9 @@ pub struct StorageCommand {
     /// Path to the `Cargo.toml` of the contract.
     #[clap(long, value_parser)]
     manifest_path: Option<PathBuf>,
+    /// Fetch the storage version of the pallet contracts (pallet version).
```

Comment:
```suggestion
    /// Fetch the storage version of the pallet contracts (state query: contracts::palletVersion()).
```

I think this makes it clearer what is being fetched.